docs/formats.md

```diff
@@ -203,20 +238,254 @@ Creates a ES6 module of the style dictionary.
 }
 ```
 
-**Example**
+**Example**  
 ```js
 export const ColorBackgroundBase = '#ffffff';
 export const ColorBackgroundAlt = '#fcfcfcfc';
 ```
 
 * * *
 
-### json
+### android/colors 
+
+
+Creates a color resource xml file with all the colors in your style dictionary.
+
+**Example**  
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+ <color name="color_base_red_5">#fffaf3f2</color>
+ <color name="color_base_red_30">#fff0cccc</color>
+ <color name="color_base_red_60">#ffe19d9c</color>
+```
+
+* * *
+
+### android/dimens 
+
+
+Creates a dimen resource xml file with all the sizes in your style dictionary.
+
+**Example**  
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+ <dimen name="size_padding_tiny">5.00dp</dimen>
+ <dimen name="size_padding_small">10.00dp</dimen>
+ <dimen name="size_padding_medium">15.00dp</dimen>
+```
+
+* * *
+
+### android/fontDimens 
+
+
+Creates a dimen resource xml file with all the font sizes in your style dictionary.
+
+**Example**  
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+ <dimen name="size_font_tiny">10.00sp</dimen>
+ <dimen name="size_font_small">13.00sp</dimen>
+ <dimen name="size_font_medium">15.00sp</dimen>
+```
+
+* * *
+
+### android/integers 
+
+
+Creates a resource xml file with all the integers in your style dictionary. It filters your
+style properties by `prop.attributes.category === 'time'`
+
+**Todo**
+
+- Update the filter on this.
+
+**Example**  
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+  <integer name="time_duration_short">1000</string>
+  <integer name="time_duration_medium">2000</string>
+  <integer name="time_duration_long">4000</string>
+```
+
+* * *
+
+### android/strings 
+
+
+Creates a resource xml file with all the strings in your style dictionary. Filters your
+style properties by `prop.attributes.category === 'content'`
+
+**Example**  
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+  <string name="content_icon_email">&#xE001;</string>
+  <string name="content_icon_chevron_down">&#xE002;</string>
+  <string name="content_icon_chevron_up">&#xE003;</string>
+```
+
+* * *
+
+### ios/macros 
+
+
+Creates an Objective-C header file with macros for style properties
+
+**Example**  
+```objectivec
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+#define ColorFontLink [UIColor colorWithRed:0.00f green:0.47f blue:0.80f alpha:1.00f]
+#define SizeFontTiny 176.00f
+```
+
+* * *
+
+### ios/plist 
+
+
+Creates an Objective-C plist file
+
+**Todo**
```

Comment:
I can work on this, in the future (I have implemented some PLIST for my iOS team in these days :) )

Reply:
Sweet! That would be excellent!

Reply:
Should we make an issue for this?